README.md

```diff
@@ -8,7 +8,65 @@ This GitHub Action takes changes from the working directory (using `git diff`) a
 ## Usage
 
 > [!IMPORTANT]
-> This GitHub Action only works on [`pull_request`](https://docs.github.com/actions/using-workflows/events-that-trigger-workflows#pull_request) events.
+> This GitHub Action works on [`pull_request`](https://docs.github.com/actions/using-workflows/events-that-trigger-workflows#pull_request) and [`pull_request_target`](https://docs.github.com/actions/using-workflows/events-that-trigger-workflows#pull_request_target) events.
+
+### Permissions
+
+This action requires write permissions for pull requests. Add the following to your workflow:
```

Comment:
This is only applicable when using the built-in `GITHUB_TOKEN` for authentication.